js/src/utils/isInShippingRates.test.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Internal dependencies
+ */
+import isInShippingRates from './isInShippingRates';
+
+describe( 'isInShippingRates', () => {
+	it( 'should return true when shippingRate exists in shippingRates, matching by id', () => {
+		const shippingRate = {
+			id: '2',
+			country: 'AU',
+			method: 'flat_rate',
+			currency: 'USD',
+			rate: 25,
+			options: [],
```

Comment:
Note: Related to the initial values mentioned in https://github.com/woocommerce/google-listings-and-ads/pull/1266#discussion_r816551941

Or maybe we could remove all `options` from this test since they don't affect the test target.

Reply:
Gotcha, I have removed them in [`fcd905b` (#1262)](https://github.com/woocommerce/google-listings-and-ads/pull/1262/commits/fcd905ba48a0478ebaf8b3d79c3f1fca40e22645).